### ISSUE TITLE
Urvi Awasthi

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#123]
Python - Thats her main langiuage she uses it everyday, advanced

ML - obviously has NLP work, has worked with classification problems in various data sets (categorical and other sets) has XGboost. Does have experience with neural networks with image processing for NLP all the time (can customize or use them out of the box)

SQL - is familiar and has used it a lot, but does not use it everyday anymore. Has taken a database course as well. Querying from databases is not a problem, this is something that she has done.

Has a full-time position; they do have schedule meetings, but overall the work is task oriented and can be available in the mornings.

What is she looking for? She is looking for more work, challenging work would be accepted.

Working at a start up Arbilex, working on the underwriting for legal. Is working on data scrapping and building the NLP solutions and then the back end on feeding those into the model.

Did work at Phizer - that roles was not as challenging - that was NLP as well

would be comfortable with the 18 hours per week, seems like her job is flexible on her work, and it also seems like shes not very challenged, her work did sign off that she can take a part -time job

I like her a lot, she seems like a go-getter and someone that wants to be challenged

**Comment by ldaudet** [comment]
Urvi Awasthi

Showed up 15 mins late, said she had the wrong time, apologized a lot

She wants to work on the side in spite of being full time because she feels like she has lots of extra time that she would like to put to productive use. After asking her fulltime, they agreed to let her do so.

She says she has 20 to 30 hours extra per week. She is also free on the weekend. She is remote so she can have flexible hours. On her calendar she has a few recurring meetings, especially MWF, but the rest is available.

She has experience with both classification and time series modelling, though more on the classification. She has done classification tens of times over, and has done a little bit of time series data for a company in the past. For the classification, she did some work in all kinds of industries, including user classification for PwC, and test classification using NLP.

She has worked with XGBoost bost for classification and regression.

She use SQL regularly as her company as they don't have data engineer yet. She says she is not an expert, but she would definitely considered familiar. She uses aggregate function and joins regularly.

When at Pfizer, she had the role of team lead on a rotating basis. They called it tech lead, and it involved building out the plans for each of the sprints, working with an Agile methodology.

At her current job, there's only two of them working on agile, and being each others team lead.

She wants to highlight her communication skills, and in particular her ability in technical communication, having worked in many different teaching related roles.

She didn't know what grid search and bayesian search where, which is very worrysome. She does the optimization of her models by trying different values for each of the parameters, and looking for elbow points on the accuracy graph.

She asked for the type of project management structure we have, because it seems that she would need technical competency.
